Exiled.API/Features/Player.cs

```diff
@@ -1528,6 +1528,21 @@ public byte GetEffectIntensity<T>()
             throw new ArgumentException("The given type is invalid.");
         }
 
+        /// <summary>
+        /// Gets all the active <see cref="PlayerEffect"/>.
+        /// </summary>
+        public List<PlayerEffect> GetAllEffects()
```

Comment:
You should return a ReadOnlyCollection<PlayerEffect> instead, since people will be able to 'modify' this list but doing so results in nothing.